modules/@apostrophecms/ui/ui/apos/components/AposTreeRows.vue

```diff
@@ -34,6 +34,7 @@
           :href="col.type === 'link' ? row[col.name] : false"
           :target="col.type === 'link' ? '_blank' : false"
           :class="getCellClasses(col, row)"
+          :disabled="getCellDisabled(col, row)"
```

Comment:
We should totally hide/remove the trash cells. The edit button is still active as well, for one. They should be gone-gone.

Reply:
We have a way to disable dragging already as well. See the line: `v-if="options.draggable && index === 0 && !row.parked"`.  Using the `parked` property would also set the `is-parked` class on it (L266), which tells vue-draggable to lock it down as well. So putting `parked: true` on the trash "page" would be the quickest way to prevent dragging. If you don't want to do that, those conditionals could also including looking for the `@apos/trash` type.

Reply:
This is still a factor @boutell